code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -690,19 +691,25 @@ var/global/list/obj/item/device/pda/PDAs = list()
 	set name = "Remove Pen"
 	set src in usr
 
+	var/obj/item/weapon/pen/Pen = locate() in src
+	var/obj/item/toy/crayon/Crayon = locate() in src

```

Comment:
This some wackass code man.

Just create a single var on the PDA item  - contained_item and then let them store only one of these four types in there.